pkg/disk/disk.go

```diff
@@ -232,21 +237,29 @@ func GlobalConfigSet(m metadata.MetadataProvider) {
 		GlobalConfigVar.DetachBeforeDelete,
 		GlobalConfigVar.ClusterID,
 	)
+	return csiCfg
+}
 
-	if GlobalConfigVar.ADControllerEnable {
-		detachConcurrency := 1
-		attachConcurrency := 1
-		if features.FunctionalMutableFeatureGate.Enabled(features.DiskParallelDetach) {
-			detachConcurrency = csiCfg.GetInt("disk-detach-concurrency", "DISK_DETACH_CONCURRENCY", 5)
-			klog.InfoS("Disk parallel detach enabled", "concurrency", detachConcurrency)
-		}
-		if features.FunctionalMutableFeatureGate.Enabled(features.DiskParallelAttach) {
-			attachConcurrency = csiCfg.GetInt("disk-attach-concurrency", "DISK_ATTACH_CONCURRENCY", 32)
-			klog.InfoS("Disk parallel attach enabled", "concurrency", attachConcurrency)
-		}
-		GlobalConfigVar.AttachDetachSlots = NewSlots(detachConcurrency, attachConcurrency)
-	} else {
-		// if ADController is not enabled, we need serial attach to recognize old disk
-		GlobalConfigVar.AttachDetachSlots = NewSlots(1, 1)
+func newBatcher(fromNode bool) (waitstatus.StatusWaiter[ecs.Disk], batcher.Batcher[ecs.Disk]) {
+	client := desc.Disk{Client: GlobalConfigVar.EcsClient}
+	ctx := context.Background()
+	interval := 1 * time.Second
+	if fromNode {
+		interval = 2 * time.Second // We have many nodes, use longer interval to avoid throttling
```

Comment:
We can optimize the interval based on distinct node names

Reply:
Sorry. I don't understand this. If batcher is initialized from csi-plugin, we cannot know how many nodes are there in the cluster. If from controller, we already have the dynamic interval mechanism from batcher to adjust the internal based on real-time load.

Reply:
Only from node, maybe we can use nodeName as random seed to generate interval time?